app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -72,4 +72,13 @@ def destroy
 
     redirect_to(wrangling_guidelines_url)
   end
-end
\ No newline at end of file
+
+   private
```

Comment:
This looks misaligned, at least on the website. Would you please check that the tabs in the file are made of 2 spaces, and that they're in the correct number?